src/dotnet-format.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
❔ Aside from limiting the set of users that can use this project, what's the advantage from this?

Reply:
System. Text. Json

Reply:
System.Text.Json is netstandard2.0. It should work without changes right?

Reply:
well so it is. I feel silly now. I don't think we should do this then until 2.1 goes out of support